catalog/core/catalog-core-directorymonitor/pom.xml

```diff
@@ -255,6 +259,11 @@
             <groupId>commons-configuration</groupId>
             <artifactId>commons-configuration</artifactId>
             <version>${commons-configuration.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-configuration2</artifactId>
+            <version>${commons-configuration2.version}</version>
```

Comment:
❓ Does it still need the old version of `commons-configuration` above?

Reply:
Yes.